src/components/FlowRunCreateForm.vue

```diff
@@ -26,6 +26,20 @@
         <p-tags-input v-model="tags" :options="deploymentTags" />
       </p-label>
 
+      <p-label label="Add Retries">
+        <p-toggle v-model="addRetries" />
+      </p-label>
```

Comment:
Personally I would remove this toggle and just show the retries all the time. I might flex the retry fields so they are formatted together as one thing. 

They're optional and having a toggle to show them doesn't provide much value imo.